commands/reshowcase

```diff
@@ -225,11 +226,62 @@ if (isBuild) {
       process.exit(1);
     })
     .then((ctx) => {
-      return ctx.serve({ port: port, servedir: outputPath });
+      return ctx.serve({
+        servedir: outputPath,
+        // ensure that esbuild doesn't take server's port
+        port: port + 1,
+      })
     })
-    .then((_serveResult) => {
-      console.timeEnd(durationLabel);
-      console.error("[Reshowcase] Watch mode started on port:", port);
+    .then(esbuildServer => {
+      const server = http.createServer((req, res) => {
+        let options = {
+          path: req.url,
+          method: req.method,
+          headers: req.headers,
+        };
+
+        const rewrite = pathRewrites?.find(rewrite => req.url.startsWith(rewrite.context))
+
+        if (rewrite) {
+          if (rewrite.socketPath) {
+            options.socketPath = rewrite.socketPath;
+            console.info(`forwarding ${req.url} to ${options.socketPath}`)
+          } else {
+            const url = new URL(rewrite.target);
+            options.host = url.hostname;
+            options.port = url.port;
+            options.headers = {
+              ...options.headers,
+              ...(rewrite.changeOrigin ? { host: `${options.host}:${options.port}` } : {})
+            };
+            console.info(`forwarding ${req.url} to ${options.host}:${options.port}`);
```

Comment:
I added these logs, but maybe it's too verbose. Probably should hide behind some flag